tests/components/test_history.py

```diff
@@ -24,13 +23,10 @@ def tearDown(self):  # pylint: disable=invalid-name
         """Stop everything that was started."""
         self.hass.stop()
 
-        db_path = self.hass.config.path(recorder.DB_FILE)
```

Comment:
I had to move the config.path() call to the class initializer because `_setup_connection` runs in a separate thread that wasn't picking up the mocked version even when I had `self.hass.start()` and `self.wait_recording_done()` in the patch context.